db/seeds.rb

```diff
@@ -88,7 +88,7 @@
                         last_name: "Mir",
                         charity_id: charity2.id,
                         kind: "Seeker",
-                        description: "Saloo is from Bhutan and was studying in university before he had to seek asylum in Britain.He needs a laptop to study social work so that he can work with his local council to support other young refugees .",
+                        description: "Saloo is from Bhutan and was studying in university before he had to seek asylum in Britain. He needs a laptop to study social work so that he can work with his local council to support other young refugees.",
```

Comment:
Maybe we can remove about what tech the seekers need from their user.description, as the requests give more information about what they need